components/styled/table/src/atoms/table.tsx

```diff
@@ -16,25 +16,37 @@ const Container = styled.div`
 const StyledTableHeader = styled.th`
   border: 1px solid rgba(200, 200, 200, 1);
 `
+export declare type ReactTableCell = string | FC<ICellProps>
```

Comment:
Very minor point but it's the convention in these projects to put interface and type declarations at the bottom of the file